staging/src/k8s.io/api/policy/v1beta1/types.go

```diff
@@ -67,7 +67,7 @@ type PodDisruptionBudgetStatus struct {
 	DisruptedPods map[string]metav1.Time `json:"disruptedPods,omitempty" protobuf:"bytes,2,rep,name=disruptedPods"`
 
 	// Number of pod disruptions that are currently allowed.
-	PodDisruptionsAllowed int32 `json:"disruptionsAllowed" protobuf:"varint,3,opt,name=disruptionsAllowed"`
+	DisruptionsAllowed int32 `json:"disruptionsAllowed" protobuf:"varint,3,opt,name=disruptionsAllowed"`
```

Comment:
FWIW this rename made this PR a fair amount bigger than it needed to be, maybe next time do this and the promotion in separate steps? This change by itself would have been mechanical and super easy to review.

Reply:
Yeah, next time I will definitely do these kind of changes in separate PRs.